crates/symbolicator/src/services/symbolication.rs

```diff
@@ -571,22 +571,24 @@ fn object_info_from_minidump_module_rust_minidump(
     ty: ObjectType,
     module: &MinidumpModule,
 ) -> RawObjectInfo {
-    let mut code_id = module.code_identifier().into_owned();
+    let mut code_id = module.code_identifier().to_string();
 
     // The processor reports an empty string as code id for MachO files
     // TODO(ja): Fix MinidumpModule::code_identifier for MachO
     if ty == ObjectType::Macho {
-        code_id = module.debug_identifier().unwrap_or_default().into_owned();
-        code_id.truncate(32); // MachO code_id is the debug_id without `0` age.
+        code_id = module
+            .debug_identifier()
+            .map(|id| id.to_string())
+            .unwrap_or_default();
     }
 
     RawObjectInfo {
         ty,
+        // TODO: shouldn't this be None if code_id is empty?
         code_id: Some(code_id),
         code_file: Some(module.code_file().into_owned()),
-        // TODO(ja): Old TODO: This should use module.id().map(_)
         // TODO(ja): This is optional now, wasn't before, check why
-        debug_id: module.debug_identifier().map(|c| c.into_owned()),
+        debug_id: module.debug_identifier().map(|id| id.to_string()),
```

Comment:
i *think* this is wrong because previously we were handling strings from breakpad while this does not use the breakpad formatting of DebugId.